### PR TITLE
Fixed typos from README.md #905

### LINKED ISSUE #905: couple of typos in readme.md
**I'm submitting a ...**
- [] bug report
- [x] feature request

### PR DESCRIPTION
**Pull request by ValentinBercaru**
Fixes #905 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Added Surge preview link

#### Changes proposed in this pull request:

- Fixed typos from README.md #905. Please check this out. :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,47 +17,47 @@ Our chat channel is on gitter : https://gitter.im/fossasia/susper.com
...
+This is a front-end for Susper running on YaCy server. The retrieval of search results is done using YaCy search API.
 
 * Solr and Javascript(JSON)
 
-Search results are displayed using Solr server which is embedded into YaCy. All search results must be provided by a YaCy search server which includes a Solr with a specialized JSON result writer. When a search request is made in one of the search templates, a http request is made to YaCy. The response is done in JSON because that can much better be parsed than XML in JavaScript. To overcome the same orgin policy in JavaScript, the result is capsuled into a JSONP response.
+Search results are displayed using Solr server which is embedded into YaCy. All search results must be provided by a YaCy search server which includes a Solr with a specialized JSON result writer. When a search request is made in one of the search templates, an HTTP request is made to YaCy. The response is done in JSON because that can much better be parsed than XML in JavaScript. To overcome the same orgin policy in JavaScript, the result is capsuled into a JSONP response.
```
Comment (harshit98) [inline README.md]: an HTTP -> a HTTP

Comment (ValentinBercaru) [reply README.md]: Sorry, but the correct form is "An HTTP".

Comment (duttaditya18) [reply README.md]: Yes! @ValentinBercaru is right.

HTTP is pronounced starting with a vowel sound (eh). So, `an` will be used.

**Comment by duttaditya18** [comment]
@ValentinBercaru Merge conflicts!

**Pull request closed without merging**